src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -95,6 +95,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String json = gson.toJson(elections);
 
         response.setContentType("application/json;");
-        response.getWriter().println(elections);
+        response.getWriter().println(json);
```

Comment:
I assume there's an implicit TODO here to write the json to DB? You can optionally leave a 
// TODO(anooshree): asdf

So that it's more clear to reviewers what's happening in follow up CLs.

Reply:
Alternatively you could also put the TODO in the PR description.

Reply:
Got it; updating code with TODOs right now.